src/Support/helpers.php

```diff
@@ -1,15 +1,16 @@
 <?php
 
+use PHLAK\SemVer\Exceptions\InvalidVersionException;
 use PHLAK\SemVer\Version;
 
 if (! function_exists('semver')) {
     /**
      * Create a SemVer version object.
      *
-     * @throws \PHLAK\SemVer\Exceptions\InvalidVersionException
+     * @throws InvalidVersionException if the provided semantic version string is invalid
```

Comment:
I'd prefer to omit the description here.

```suggestion
     * @throws InvalidVersionException
```

Reply:
I think complete PHPDoc is good practice. You have to decide. 🙃